src/applications/vre/28-1900/config/chapters/communication-preferences/communicationPreferences.js

```diff
@@ -80,18 +80,21 @@ export const uiSchema = {
       'ui:title': 'Mornings 6:00 to 10:00 a.m.',
       'ui:options': {
         hideEmptyValueInReview: true,
+        ariaDescribedBy: 'appointment-time-desc',
```

Comment:
@joshkimux - because each of these checkboxes renders as its own field, each one will announce the aria-describedby element.